editor/src/components/canvas/canvas-strategies/drag-to-insert-strategy.tsx

```diff
@@ -194,8 +196,9 @@ function runTargetStrategiesForFreshlyInsertedElement(
   editorState: EditorState,
   strategyState: StrategyState,
   interactionState: InteractionSession,
-  commandLifecycle: 'mid-interaction' | 'end-interaction',
+  commandLifecycle: InteractionLifecycle,
   insertionSubjects: Array<{ command: InsertElementInsertionSubject; frame: CanvasRectangle }>,
+  strategyLifeCycle: InteractionLifecycle,
```

Comment:
oh this will break my brain the next time I read this code :D I understand the purpose of the two lifecycles in this situation, but somehow their name is not distinct enough... 🤔 
But I don't have a better name to propose

Reply:
More consistent naming in 5e101ff7b481769539deeba0b2bda173ff843195